### DISCUSSION TITLE
关于“ABSTRACT”替换为“Abstract”、目录页面添加引导线

### DISCUSSION THREAD

**Discussion by Linkzhuu** [nju-lug/NJUThesis#109] [General]
研究生模版，学院格式普遍使用“Abstract”、引导线，请问如何修改
<img width="480" alt="截屏2022-03-28 上午1 02 23" src="https://user-images.githubusercontent.com/47322049/160292401-b7882e2a-f7aa-4449-84d9-67b9d8974e59.png">

**Comment by atxy-blip** [comment]
### 使用小写字母

源码在这里定义摘要用的名称：
```
\clist_map_inline:nn
  {
    { abstract   } { 摘要       } { ABSTRACT            },
    % 用于书签的摘要名称字样
    { abstract_a } { 中文摘要   } { ABSTRACT            },
    % 用于小标题的摘要名称字样
    { abstract_b } { 摘\qquad{}要 } { ABSTRACT          },
```

目前修改方法是 `njuthesis-user-v0.16.*.zip` 中复制一份 `njuthesis-graduate.def` 放在当前项目目录下，然后修改这个文件里面 `abstract_a` 后面的英文名称。

### 引导线

有人问过。不加引导线是 LaTeX 标准文档类的配置方法，一个可行的修改手段见 <https://github.com/nju-lug/NJUThesis/issues/66>。

-----

有空的时候再看看能不能改简单一点……

**Reply by Linkzhuu** [reply]
感谢🙏
